app/views/krikri/qa_reports/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<% # krikri/qa_reports/index %>
+
+<h1>QA Reports</h1>
```

Comment:
from here downward should be wrapped in `<div id='main-container' class='container'>...</div>`